packages/hub/model.js

```diff
@@ -12,6 +12,20 @@ module.exports = class Model {
     return priv.get(this).contentType;
   }
 
+  getId() {
+    let { jsonapiDoc } = priv.get(this);
+    if (jsonapiDoc) {
+      return jsonapiDoc.id;
+    }
+  }
```

Comment:
I guess this exists because `getField('id')` is async, and id doesn't ever really need to be async?

Reply:
I was just assuming that `id` was not a field and using an accessor for it. but now that I look at Model more closely I see that getField can handle this. I'll clean this up.

Reply:
and also to be consistent, we should probably take the same approach for `meta`